internal/lib/types.go

```diff
@@ -161,18 +152,34 @@ func (s *ContainerCertificationSubmitter) Submit(ctx context.Context) error {
 	}
 	defer logfile.Close()
 
+	options := make([]pyxis.CertificationInputOption, 0, 3)
```

Comment:
Initialize this in one statement. There is no reason to do the append like this.

Reply:
Fixed.